modules/javafx.graphics/src/main/java/com/sun/glass/ui/Platform.java

```diff
@@ -62,15 +64,13 @@ else if (userPlatform.equals("ios"))
                 return type;
             }
 
-            String osName = System.getProperty("os.name");
-            String osNameLowerCase = osName.toLowerCase(Locale.ROOT);
-            if (osNameLowerCase.startsWith("mac") || osNameLowerCase.startsWith("darwin")) {
+            if (PlatformUtil.isMac()) {
```

Comment:
this change might be ok, but it is not strictly equivalent.
before, MAC path would be selected if os.name lowercase starts from either "mac" or "darwin", after the change, only starts with "Mac".
Do we want to support "darwin"?

Reply:
I'd still prefer to use PlatformUtil, since this is a cleaner solution that the variety of methods used before:
```
osNameLowerCase.startsWith("wind"
"ios".equals(System.getProperty("os.name")
osName.startsWith("Windows"
osName.startsWith("Mac OS"
("os.name").toLowerCase().indexOf("win")
```

Reply:
I initially had the same thought. What this actually highlights, though, is the danger of having this sort of check scattered in multiple places, but done differently. If it was important that "darwin" be recognized (it isn't), then it would be wrong to have this check recognize it, but all other "isMac" checks not recognize it.

Reply:
Do we officially support Darwin OS?

Reply:
Wikipedia says it's currently at the level of macOS Ventura

22.4.0 | March 27, 2023 | macOS 13.3, iOS 16.4, iPadOS 16.4, watchOS 9.4 and tvOS 16.4

https://en.wikipedia.org/wiki/Darwin_(operating_system)#Darwin_20_onwards

Reply:
Darwin is the kernel name of macOS/OS X, so technically when we speak of Darwin we speak of macOS. My guess for what the goal was here is to support some older naming where `os.name` prop had what `uname` command returns (which returns what kernel reports, which is its name "Darwin"). So, technically, we do support it because we support macOS?

Reply:
Darwin as its own OS stopped existing after 2002 (at least german wilipedia sais so)

Reply:
Apple still calls their macOS Kernel "Darwin", but that's not the important bit. If it was _ever_ reported as the value of `os.name` (I'm not sure it was), it would have been in the pre-Java 6 time frame. It's a long-dead holdover.